doc/nrf/test_and_optimize/optimizing/power_general.rst

```diff
@@ -181,11 +181,41 @@ Bluetooth Mesh
 
 The Bluetooth Mesh protocol offers the :ref:`ug_bt_mesh_configuring_lpn` feature for optimizing the power consumption of the Bluetooth Mesh devices.
 
+Gazell
+======
+
+Gazell is an asymmetrical protocol where the Device role is power-optimized.
```

Comment:
Is it possible to add such a point? It is how wireless keyboard and mouse conserve battery in real life.

A Gazell device attains lowest power consumption in idle times.